### ISSUE TITLE
add decision_function to MLPClassifier

### ISSUE THREAD

**Issue by mheilman** [civisanalytics/muffnn#67]
It'd be nice to have a `decision_function` in the MLP classifier that returns logits [like sklearn's LogisticRegression](http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html#sklearn.linear_model.LogisticRegression.decision_function), etc.  This might lead to better performance if the MLPClassifier is used in stacking.

**Comment by beckermr** [comment]
To add details here, I think the idea is to output the final logits before normalization.
